pkgs/development/python-modules/pydantic-core/default.nix

```diff
@@ -0,0 +1,79 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, cargo
+, rustPlatform
+, rustc
+, typing-extensions
+, pythonRelaxDepsHook
+, pytestCheckHook
+, hypothesis
+, pytest-benchmark
+, pytest-timeout
+, pytest-mock
+, dirty-equals
+}:
+
+buildPythonPackage rec {
+  pname = "pydantic-core";
+  version = "2.1.3";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "pydantic";
+    repo = "pydantic-core";
+    rev = "v${version}";
+    hash = "sha256-JdwzCD7ppxfqf/KPMDtCyw+XaZOzyKKgd7AhtF7gVKg=";
+  };
+
+  cargoDeps = rustPlatform.importCargoLock {
+    lockFile = ./Cargo.lock;
+  };
+
+  nativeBuildInputs = [
+    cargo
+    rustPlatform.cargoSetupHook
+    rustPlatform.maturinBuildHook
+    rustc
+    typing-extensions
+    pythonRelaxDepsHook
+  ];
+
+  propagatedBuildInputs = [
+    typing-extensions
+  ];
+
+  pythonRelaxDeps = [
+    # NOTE: bumping typing-extensions to 4.7.1 would
```

Comment:
We usually don't comment those things at all

Reply:
Removed in [force-push](https://github.com/NixOS/nixpkgs/compare/951c4557822991b5c27b5d4819a091b9b956618a..4902a0946c02feef94b1a3b5578a13a611925694)